### PR TITLE
Added test cases for CreateDiscovery and LoadTlsConfig

### PR DESCRIPTION
**Pull request by rajdeepd**
Signed-off-by: Rajdeep Dua <dua_rajdeep@yahoo.com>

### CODE REVIEW

**Comment by rajdeepd** [comment]
@vieux @aluzzardi kindly review

**Comment by abronan** [comment]
Thanks @rajdeepd!

Probably a doubt on where to put the fake certs for testing (as we probably need those in other places), so we'd probably move the certs out of cli in a specific directory dedicated to `tls` tests and certs generation. I'll let @aluzzardi and @vieux express their preference.

LGTM

**Comment by rajdeepd** [comment]
Can we get this in a discuss doing another pull request to change the location

**Comment by rajdeepd** [comment]
Hi guys @aluzzardi , @vieux kindly review..

File: cli/manage_test.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"flag"
+	mcli "github.com/codegangsta/cli"
```
Comment (vieux) [inline cli/manage_test.go]: can you please not alias this to `mcli` so it's consistant with the rest of the codebase ? Thanks

File: cli/test-tls/ca.pem [hunk 2]
```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```
Comment (aluzzardi) [inline cli/test-tls/ca.pem]: Could you move these to `cli/testdata/tls/`?

We have a convention for test data (e.g. `test/integration/testdata`)

**Comment by abronan** [comment]
Hi @rajdeepd, sorry for the delay of the initial review, any status on this and on addressing the comments above? Thanks!

**Comment by vieux** [comment]
Please comment if you want us to reopen the PR

**Comment by Yam408** [comment]
sorry guys if I make some sick. I have never any experience or ideas. I'm still training &  learn English language then almost makes wrong steps. 
I'll be careful as I can & follo advice . just your support and patience friends. thanks

**Pull request closed without merging**